nodes/node.go

```diff
@@ -10,19 +10,18 @@ import (
 	"github.com/stampzilla/gozwave/database"
 	"github.com/stampzilla/gozwave/events"
 	"github.com/stampzilla/gozwave/functions"
-	"github.com/stampzilla/gozwave/serialapi"
+	"github.com/stampzilla/gozwave/interfaces"
 )
 
-func New(address int, connection *serialapi.Connection, eventQue chan interface{}) *Node {
+func New(address int) *Node {
```

Comment:
Code readability is reason enough, but there's a bonus: it lets you put methods on the type later.

One thing you might want to do is pack the HomeID (uint32) and DeviceAddress (byte) together into a:

```go
// Address is a fully-qualified ZWave device address.
// It contains both the Home ID and Node ID packed together.
type Address uint64
```

Then you can have a String method, Node accessor, HomeID accessor, etc.

And then you're ready later for a system with multiple controllers on different HomeIDs.